app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```

Comment:
I think it's cleaner to keep all the 'static' pages in one controller, or have a separate controller for every page - but a mix isn't ideal.
Let's put everything in home, for now, since that's what we've been doing, so far.